Source/JavaScriptCore/runtime/ClassInfo.h

```diff
@@ -26,6 +26,12 @@
 #include "JSCast.h"
 #include <wtf/PtrTag.h>
 
+#if HAVE(36BIT_ADDRESS)
+#define CLASS_INFO_16BYTE_ALIGNMENT alignas(16)
```

Comment:
This should be called `CLASS_INFO_ALIGNMENT` since it's not always 16 byte aligned.

Reply:
Updated